sdk/purview/azure-purview-sharing/README.md

```diff
@@ -0,0 +1,182 @@
+# Azure Share client library for Python
+
+Microsoft Purview Share is a fully managed cloud service.
+
+**Please rely heavily on the [service's documentation][sharing_product_documentation] and our [protocol client docs][protocol_client_quickstart] to use this library**
+
+[Source code][source_code] | [Package (PyPI)][client_pypi_package] | [Product documentation][sharing_product_documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Purview Sharing client library for Python with [pip][pip]:
+
+```bash
+pip install azure-purview-sharing
+```
+
+### Prerequisites
+
+- You must have an [Azure subscription][azure_subscription] and a [Purview resource][purview_resource] to use this package.
+- Python 3.6 or later is required to use this package.
+
+### Authenticate the client
+
+#### Using Azure Active Directory
+
+This document demonstrates using [DefaultAzureCredential][default_cred_ref] to authenticate via Azure Active Directory. However, any of the credentials offered by the [Azure.Identity][azure_identity] will be accepted.  See the [Azure.Identity][azure_identity] documentation for more information about other credentials.
+
+Once you have chosen and configured your credential, you can create instances of the `PurviewSharing`.
+
+```python
+from azure.purview.sharing import PurviewSharing
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+client = PurviewSharing(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+```
+
+## Key concepts
+
+## Examples
+
+The following section shows you how to initialize and authenticate your client and share data.
+
+### Create sent share
+
+```python Snippet:create_a_sent_share
```

Comment:
you have to write the snippets out. Make sure the snippets are fully copy-pastable, meaning they include the client initialization etc so they are able to run once users sub in their own credentials